README.md

```diff
@@ -70,10 +70,13 @@ This repo hosts:
 [Make your own badges!][custom badges]
 (Quick example: `https://img.shields.io/badge/left-right-f39f37`)
 
-Browse a [complete list of badges][shields.io] and find the badge you want to add to your GitHub readmes or other web page. Click on the badge to fill in required data (like server, colors etc.), and its ready for use!
-
 [custom badges]: http://shields.io/#your-badge
 
+### Quickstart
+Browse a [complete list of badges][shields.io] and locate a particular badge by using the search bar or by browsing the categories. Click on the badge to fill in required data elements for that badge type (like your username or repo) and optionally customize (label, colors etc.). And it's ready for use! 
```

Comment:
minor tweak to the end, but otherwise LGTM
```suggestion
Browse a [complete list of badges][shields.io] and locate a particular badge by using the search bar or by browsing the categories. Click on the badge to fill in required data elements for that badge type (like your username or repo) and optionally customize (label, colors etc.). Then it's ready for use! 
```